ompi/debuggers/predefined_gap_test.c

```diff
@@ -145,6 +146,7 @@ int main(int argc, char **argv) {
     GAP_CHECK("f_flags", test_file, f_flags, f_amode,  1);
     GAP_CHECK("f_f_to_c_index", test_file, f_f_to_c_index, f_flags, 1);
     GAP_CHECK("error_handler", test_file, error_handler, f_f_to_c_index, 1);
+    GAP_CHECK("errhandler_type", test_file, errhandler_type, error_handler, 1);
```

Comment:
and here too
`GAP_CHECK("f_io_version", test_file, f_io_version, errhandler_type, 1);`

Reply:
added